INSTALL.md

```diff
@@ -57,22 +57,23 @@ Also:
 | RPM packages via "make rpm"                     | 4.11 or later  | rpm                     | rpm                     | (n/a)                   |
 | unit tests                                      | 1.1.0 or later | libcmocka-devel         | libcmocka-devel         | libcmocka-dev           |
 
-## Optional testing dependencies
+## Optional Testing Dependencies
 * procps and psmisc (if running cts-exec, cts-fencing, or CTS)
 * valgrind (if running CTS valgrind tests)
-* python3-systemd (if using CTS on cluster nodes running systemd)
+* python3-dateutil and python3-systemd (if using CTS on cluster nodes running
+  systemd)
```

Comment:
While we're here, it's now more correct to say "CTS lab"

Reply:
updated